cve_bin_tool/checkers/openswan.py

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/python3
+
+"""
+CVE checker for openswan
+
+https://www.cvedetails.com/product/57217/Xelerance-Openswan.html?vendor_id=20146
+
+"""
+import os
+from ..util import regex_find
+
+
+def get_version(lines, filename):
+    """returns version information for openswan as found in a given file.
+    The version info is returned as a tuple:
+        [modulename, is_or_contains, version]
+
+    modulename will be openswan if openswan is found (and blank otherwise)
+    is_or_contains indicates if the file is a copy of openswan or contains one
+    version gives the actual version number
+
+    VPkg: xelerance, openswan
+    """
+    regex = [r"Openswan ([0-9]+\.[0-9]+\.[0-9]+)"]
+    version_info = dict()
+    version = regex_find(lines, *regex)
+
+    if "whack" in filename:
```

Comment:
I see a lot more options than just "whack" in https://github.com/xelerance/Openswan/tree/master/programs -- are the others not included because of risks of false positives?  Something else?

Reply:
I have checked all the files in https://github.com/xelerance/Openswan/tree/master/programs and 
added those which have the strings we are looking for.